### PR TITLE
Consolidate setup docs

### PR DESCRIPTION
**Pull request by ZoydSportello**
# Summary

This PR consolidates the setup instructions for both binaries and Docker into a single file, `setup-axelar.md`. Where needed, this file distinguishes steps for Docker vs. binaries setup. 

I removed all examples of output to future proof the doc--they're probably going to change a lot and having outdated examples can erode trust. Unless the response examples can be automatically generated from the source code, I'd recommend for not including them until they can be guaranteed to be relatively stable.  

This PR does not change any TOC configuration, which may be needed to successfully replace the two old files with the new, consolidated file. 

Here are some notes and questions for next steps and possible future improvements and clarifications:

- Use tabs for steps where there are differences b/t setup methods—Docker or binaries, especially for the requirements section. Then you’d only see the stuff about Docker images etc if you’re using and choosing Docker
- Make all of the L2 sections collapsible
- Make the part about Ethereum accounts into an aside or something similar
- Clarify the Ethereum accounts section. I don’t totally understand the import of that. Do we expect or want users to send transactions from ETH to Axelar? Is it just that they need to sign those with Axelar’s keys?
- In the “  [Generate a key on Axelar and get test tokens](https://docs.axelar.dev/#/setup/setup-with-docker?id=generate-a-key-on-axelar-and-get-test-tokens) “ section, why do you need to find the address of the validator if it’s returned from the setup script?
- Similar—when you create the validator, are you also creating a key then? IS that a validator key and then you use that to create a key for a different kind of node?
- How do you make a node into an “official” validator?

@ggutoski fyi

### CODE REVIEW

**Comment by ggutoski** [comment]
Thank you!

> Clarify the Ethereum accounts section. I don’t totally understand the import of that. Do we expect or want users to send transactions from ETH to Axelar? Is it just that they need to sign those with Axelar’s keys?

The section on Ethereum accounts is a complete non sequitur and should be removed from the setup doc.  Our fault.

> In the “  [Generate a key on Axelar and get test tokens](https://docs.axelar.dev/#/setup/setup-with-docker?id=generate-a-key-on-axelar-and-get-test-tokens) “ section, why do you need to find the address of the validator if it’s returned from the setup script?

Good point.  One could argue that script output might not always be available and `axelard keys show` is a nice self-contained way to get the address.  But at present I agree that this step only adds complexity to this particular doc.

> Similar—when you create the validator, are you also creating a key then? IS that a validator key and then you use that to create a key for a different kind of node?

Becoming a validator is another process that is out of scope of this doc.  Happy to chat about it though if you're curious.  In short, yes more keys are created when you become a validator.  It's unfortunate that `validator` is the default name for the first account you create when you spin up a new non-validating node.  I think that default inherited from Cosmos SDK and we should consider changing it.

> How do you make a node into an “official” validator?

https://docs.axelar.dev/#/validator-zone/setup/overview That's out of scope for now but feel free to take a look anyway. 😉

File: docs/setup/setup-axelar.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+To monitor the log file in real time, open a new terminal window and run:
+
+```bash
+tail -f testnet.log
+```
+
+To filter the log to make finding relevant information easier, run:
+
+```bash
+docker logs -f axelar-core 2>&1 | grep -a -e threshold -e num_txs -e proxies
```
Comment (ZoydSportello) [inline docs/setup/setup-axelar.md:65]: Future improvement: explain why you're grepping for these terms. It looks like transaction tallies and threshold and proxy events are important--why?

File: docs/setup/setup-axelar.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+  To kill this process:
+
+      kill -9 $(pgrep -f 'axelard start')
+
+- `vald` -- Broadcasts chain event transactions. 
+
+  To kill this process:
+
+      kill -9 $(pgrep -f 'axelard vald-start')
```
Comment (ZoydSportello) [inline docs/setup/setup-axelar.md:202]: Is this right? Do you have to grep for `axelard` to kill `vald`?

File: docs/setup/setup-axelar.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+- `tofnd` -- Handles threshold crypto operations.
+
+  To kill this process:
+
+      kill -9 $(pgrep tofnd)
```
Comment (ZoydSportello) [inline docs/setup/setup-axelar.md:208]: Should this be:

```suggestion
      kill -9 $(pgrep -f `tofnd`)
```

File: docs/setup/setup-axelar.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+
+- `tofnd` -- Handles threshold crypto operations.
+
+You can stop or remove these containers by running:
+
+```bash
+docker stop axelar-core vald tofnd
+```
+
+If you see an error related to insufficient gas at any point during the workflow, add these flags:
```
Comment (ZoydSportello) [inline docs/setup/setup-axelar.md:182]: > add these flags

To what command?

**Comment by ZoydSportello** [comment]
Thanks for the answers, Gus! I tried to incorporate them into this revision--removed the Eth section and added a couple of links to the validator docs. 

Also tried to fix the formatting of the indented code samples.

**Comment by ZoydSportello** [comment]
Re: tabs, I had hoped that it would be easy to implement something like these [docusaurus tabs](https://docusaurus.io/docs/markdown-features/tabs#customizing-tab-headings)--but I'm not sure how or if Docsify supports that. 

Not necessarily for this PR but eventually something like this [includes library](https://github.com/simologos/docusaurus-plugin-includes) would likely be really useful, too.

File: docs/setup/setup-axelar.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+You can stop or remove these containers by running the following commands:
+
+- **Docker**
+
+      docker stop <CONTAINER_NAME>
+
+- **Binaries**
+
+      kill -9 $(pgrep -f '<PROCESS_NAME>')
```
Comment (ZoydSportello) [inline docs/setup/setup-axelar.md:174]: Can't tell from the existing doc if it's actually this straightforward of if there are some weird nuances for these processes.

**Comment by ZoydSportello** [comment]
>  It's unfortunate that validator is the default name for the first account you create when you spin up a new non-validating node. I think that default inherited from Cosmos SDK and we should consider changing it.

Yeah, the old three hardest things in computer science: naming and off by one errors 😄. Speaking from experience, it's good to adjust these things as early as possible. Else, the docs and tooling can get to be kinda tortured and hacky.

**Pull request closed without merging**